src/assets/wise5/authoringTool/peer-grouping/select-peer-grouping-option/select-peer-grouping-option.component.ts

```diff
@@ -35,9 +35,13 @@ export class SelectPeerGroupingOptionComponent implements OnInit {
   }
 
   setPeerGroupingLogicName(): void {
-    this.peerGroupingLogicName = availableLogic.find(
-      (logic) => logic.value === this.peerGrouping.logic
-    ).name;
+    if (new RegExp(DIFFERENT_IDEAS_REGEX).exec(this.peerGrouping.logic) != null) {
+      this.peerGroupingLogicName = DIFFERENT_IDEAS_NAME;
+    } else {
+      this.peerGroupingLogicName = availableLogic.find(
+        (logic) => this.peerGrouping.logic === logic.value
```

Comment:
Put changing variable on the left side of the comparator.
```suggestion
        (logic) => logic.value === this.peerGrouping.logic;
```